tests/js/setup-globals.js

```diff
@@ -43,6 +43,14 @@ global._googlesitekitLegacyData = {
 	},
 };
 
+// eslint-disable-next-line no-undef
```

Comment:
This exclusion doesn't seem to be necessary (maybe from during dev?).
```suggestion
```

Reply:
This still needs to be removed.